### PR TITLE
KEYCLOAK-16195 [Quickstarts README] app-profile-saml-jee-jsp

### PR DESCRIPTION
**Pull request by Pepo48**
* edited unclear realm import part in README.md

Signed-off-by: Peter Zaoral <pzaoral@redhat.com>

@andymunro PTAL once you got a chance. Thanks!
Cc: @sguilhen

### CODE REVIEW

**Comment by Pepo48** [comment]
@andymunro just for the sake of clarification, in each of the related JIRAs it is mentioned another problem related to 'Keycloak' name occurrence (instead of 'RH SSO') across the whole prod quickstart repo. None of these PRs that I just sent is fixing this. It's a higher-level issue with the productize.sh script. Therefore I created 'an umbrella' JIRA which will fix this for all quickstarts at once. It was discussed with @pskopek and the fix is pranned for 7.5.x.

https://issues.redhat.com/browse/KEYCLOAK-16198

File: app-profile-saml-jee-jsp/README.md [hunk 1]
```diff
@@ -59,8 +59,9 @@ Finally you need to configure the adapter, this is done by retrieving the adapte
 * Move the file `keycloak-saml.xml` to the `config/` directory in the root of the quickstart
 
 As an alternative you can create the client by importing the file [client-import.json](config/client-import.json) and
-copying [config/keycloak-saml-example.xml](config/keycloak-saml-example.xml) to `config/keycloak-saml.xml`. Finally,
-you need to edit `config/keycloak-saml.xml` and replace `REPLACE WITH REALM CERTIFICATE` with the actual realm certificate.
+copying [config/keycloak-saml-example.xml](config/keycloak-saml-example.xml) to `config/keycloak-saml.xml`. 
+In case you work with a realm created by your own, you need to edit `config/keycloak-saml.xml`
```
Comment (andymunro) [inline app-profile-saml-jee-jsp/README.md:63]: change "a realm created by your own" to "a realm that you created"

**Review by andymunro — CHANGES_REQUESTED** [review]
Just one change requested. Thanks.

File: app-profile-saml-jee-jsp/README.md (hunk 1 above)
Comment (Pepo48) [reply app-profile-saml-jee-jsp/README.md]: Changed. Thanks @andymunro

**Review by andymunro — APPROVED** [review]
Looks good. Thanks.

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**